src/frontend/src/pages/WorkPackageForm/WorkPackageFormView.tsx

```diff
@@ -154,23 +144,29 @@ const WorkPackageEditContainer: React.FC<WorkPackageEditContainerProps> = ({ wor
         stage
       };
       await mutateAsync(payload);
-      exitEditMode();
+      exitActiveMode();
     } catch (e) {
       if (e instanceof Error) {
         toast.error(e.message);
         return;
       }
     }
   };
-  const projectWbsString: string = projectWbsPipe(workPackage.wbsNum);
+  const projectWbsString: string = projectWbsPipe(wbsElement.wbsNum);
 
   return (
     <PageLayout
-      title={`${wbsPipe(workPackage.wbsNum)} - ${workPackage.name}`}
+      title={`${isProject(wbsElement.wbsNum) ? 'New Work Package' : wbsPipe(wbsElement.wbsNum)} - ${wbsElement.name}`}
       previousPages={[
         { name: 'Projects', route: routes.PROJECTS },
-        { name: `${projectWbsString} - ${workPackage.projectName}`, route: `${routes.PROJECTS}/${projectWbsString}` }
+        {
+          name: `${projectWbsString} - ${
+            isProject(wbsElement.wbsNum) ? wbsElement.name : (wbsElement as WorkPackage).projectName
+          }`,
+          route: `${routes.PROJECTS}/${projectWbsString}`
+        }
       ]}
+      headerRight={<ChangeRequestDropdown control={control} name="crId" errors={errors} />}
```

Comment:
remove this one

Reply:
can you also put the form submit/cancel buttons here like Jack has in his PR. Also if you can make the page header sticky (basically you can scroll through the rest of the form without the title/submit-cancel buttons moving), that would be cool. I think it should be easy, but if it's super tricky, not a big priority